packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -163,10 +163,13 @@
 }
 
 .mdc-textfield--box {
+  @include mdc-text-field-corner-radius-all(0);
```

Comment:
Is it really necessary to have 2 separate mixins for this? It is possible to specify the 4 corners separately with `border-radius` as well, which would also avoid needing to define 3 styles instead of one here.

Reply:
+1, I think we should just remove mdc-text-field-corner-radius-all....its not necessary for the feature request, and it doesn't provide much utility.

Reply:
Actually I was arguing the opposite - the `all` mixin (maybe without the `-all` in the name) is the one thing we need, because I think it should be feasible to send it anything that will work with `border-radius`, whether that be 1 value or 4.